docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -368,7 +384,7 @@ s| @DenyAll | Specifies that no security roles are allowed to invoke the specifi
 s| @PermitAll | Specifies that all security roles are allowed to invoke the specified methods.
 
 `@PermitAll` lets everybody in, even without authentication.
-s| @RolesAllowed | Specifies the list of security roles permitted to access methods in an application.
+s| @RolesAllowed | Specifies the list of security roles authorized to access methods in an application.
```

Comment:
```suggestion
s| @RolesAllowed | Specifies the list of security roles allowed to access methods in an application.
```

Reply:
`authorized` may be read by some users as if the authorization decision has been made, so may be `allowed` would be better, `allowed` is already used just above as well